nifi-nar-bundles/nifi-slack-bundle/nifi-slack-processors/src/main/java/org/apache/nifi/processors/slack/PostSlack.java

```diff
@@ -180,6 +188,13 @@ public class PostSlack extends AbstractProcessor {
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .build();
 
+    public static final PropertyDescriptor SSL_CONTEXT_SERVICE = new PropertyDescriptor.Builder()
+            .name("SSL Context Service")
+            .description("If specified, indicates the SSL Context Service that is used to communicate with the "
+                    + "remote server. If not specified, communications will not be encrypted")
```

Comment:
This description is not correct, in absence of the `SSL Context Service`, communication will be encrypted, but the Processor will use the system default certificate authorities.